src/main/java/seedu/address/storage/JsonAdaptedHouse.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.house.Block;
+import seedu.address.model.house.House;
+import seedu.address.model.house.Landed;
+import seedu.address.model.house.Level;
+import seedu.address.model.house.NonLanded;
+import seedu.address.model.house.PostalCode;
+import seedu.address.model.house.Street;
+import seedu.address.model.house.UnitNumber;
+
+/**
+ * Jackson-friendly version of {@link House}
+ */
+public class JsonAdaptedHouse {
+    private final String block;
+    private final String level;
+    private final String postalCode;
+    private final String street;
+    private final String unitNumber;
+    private final String type; // "NonLanded" or "Landed": This is different from housingType.
```

Comment:
umm, shall we store them as Landed/HDB/Condominium instead of Nonlanded?

Reply:
for next iteration

Reply:
Yes, we should, that one is actually meant to be HDB/Condo/Landed, but now the housingType is under House, which we need to take it out and replace type with HousingType